Content.Server/Body/Components/RespiratorComponent.cs

```diff
@@ -32,13 +32,13 @@ public sealed class RespiratorComponent : Component
 
         // TODO HYPEROXIA?
 
-        [DataField("damage", required: true)]
+        [DataField("damage")]
         [ViewVariables(VVAccess.ReadWrite)]
-        public DamageSpecifier Damage = default!;
+        public DamageSpecifier Damage = new ();
 
-        [DataField("damageRecovery", required: true)]
+        [DataField("damageRecovery")]
         [ViewVariables(VVAccess.ReadWrite)]
-        public DamageSpecifier DamageRecovery = default!;
+        public DamageSpecifier DamageRecovery = new ();
```

Comment:
these should be required still

Reply:
im p sure there were tests that dont have these defined which worked, so i figured they are fine with a default. what should the test have as values then?

Reply:
just copy them from the human prototype